lib/WUI/wui_api.h

```diff
@@ -48,6 +48,7 @@ typedef enum {
     ETHVAR_DNS1_IP4, // ip_addr_t, dns1_ip4
     ETHVAR_DNS2_IP4, // ip_addr_t, dns2_ip4
     ETHVAR_MAC_ADDRESS, // is not included in ethconfig (used in stringifying for screen)
+    ETHVAR_NTP_ADDRESS, // char[256+1], hostname or ip
```

Comment:
256 seems like a lot.

Reply:
I need to update the comment as it is now 60+1 in fact.

Reply:
Updated the comment